systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -54,11 +56,12 @@
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @ExtendWith({TestExecutionWatcher.class})
 @DisplayNameGeneration(IndicativeSentences.class)
+@io.skodjob.testframe.annotations.ResourceManager
```

Comment:
So currently we would have both:
```java
@io.skodjob.testframe.annotations.ResourceManager
and 
protected final ResourceManager resourceManager = ResourceManager.getInstance();
```
Eventually, we would remove `protected final ResourceManager resourceManager = ResourceManager.getInstance()` when we integrate other stuff around our CRDs and other resources, right?

Reply:
Yeah as I mentioned in the previous comment from Korny, I'm doing it in small pieces that will be actually reviewed by someone :) otherwise yeah I can just remove everything and have 20k+ lines changed :D.
So now, I keep both, but I will remove it in next PR I guess.

Reply:
I think that I can remove it now, but yeah, it's not last PR to the `test-frame` branch and IDK how much things I will have to actually fix more, which can lead to a lot of lines changed. So yes, eventually we will get rid of the whole ResourceManager class and we will keep just the implementation from test-frame, but it will not be in this PR.